config.go

```diff
@@ -67,4 +67,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Labels holds runtime-specified information that is not structured by this specification
+	Labels map[string]json.RawMessage `json:"labels"`
```

Comment:
sorry, but why `json.RawMessage`? I presume for holding structures instead of just a `string`, but couldn't that be accomplished with a `map[string]interface{}` ?